CCExtractor/ultimate_alarm_clock
# Feature: Multi-language Support

### Description
This pull request introduces multi-language support to the Ultimate Alarm Clock project. The primary goal is to make the application accessible to a wider audience by providing support for multiple languages. The key changes include the addition of language files for German, Russian, French, and Spanish, as well as modifications to the UI to display content in the user's preferred language.

### Proposed Changes
Added language file called language.dart containing translations for German (de_DE), Russian (ru_RU), French (fr_FR), and Spanish (es_ES).
Modified UI elements to use translation keys for dynamic content.
Translated key UI elements into German, Russian, French, and Spanish.

## Screenshots
<img width="200" alt="Screenshot 2023-12-31 200815" src="https://github.com/CCExtractor/ultimate_alarm_clock/assets/85829910/ebbeb889-9520-4fdd-9e14-db6d7b27f215">
<img width="197" alt="Screenshot 2023-12-31 200934" src="https://github.com/CCExtractor/ultimate_alarm_clock/assets/85829910/f5e1ed5f-6016-424a-a292-5691555f7328">
<img width="196" alt="Screenshot 2023-12-31 201022" src="https://github.com/CCExtractor/ultimate_alarm_clock/assets/85829910/8b6e99c3-d7cf-4190-9033-dfd0ae1d4cab">

## Checklist
[ ] Tests have been added or updated to cover the changes
[ ] Documentation has been updated to reflect the changes
[ ] Code follows the established coding style guidelines
[ ] All tests are passing

### How to Test
Clone this branch and run the application.
Navigate to the language settings and change the language to German, Russian, French, or Spanish.
Verify that UI elements are displayed in the selected language.
Test other functionalities to ensure that multi-language support does not affect the app's behavior negatively.

Checklist
 Added translations for key UI elements in language files.
 Updated UI elements to use translation keys.
 Tested the application for language changes.
 Updated documentation (if applicable).

Reviewers
@MarkisDev @cfsmp3 

Closing Note
Thank you for reviewing this pull request. I look forward to any feedback and suggestions for improvement.
